pom.xml

```diff
@@ -1,171 +1,261 @@
-            <testTarget>1.8</testTarget>
-          </configuration>
-        </plugin>
-      </plugins>
-    </pluginManagement>
-  </build>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.2.4.RELEASE</version>
+		<relativePath/>
+	</parent>
+
+	<groupId>eu.europeana.metis</groupId>
+	<artifactId>metis-sandbox</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>metis-sandbox</name>
+	<description>Metis sandbox pilot</description>
+
+	<!-- Github connection -->
+	<scm>
+		<connection>scm:git:https://github.com/europeana/metis-sandbox</connection>
+		<url>https://github.com/europeana/metis-sandbox</url>
+		<tag>HEAD</tag>
+		<developerConnection>scm:git:https://github.com/europeana/metis-sandbox</developerConnection>
+	</scm>
+
+	<distributionManagement>
+		<repository>
+			<id>artifactory.eanadev.org</id>
+			<name>artifactory.eanadev.org-releases</name>
+			<url>http://artifactory.eanadev.org/artifactory/libs-release-local</url>
+		</repository>
+		<snapshotRepository>
+			<id>artifactory.eanadev.org</id>
+			<name>artifactory.eanadev.org-snapshots</name>
+			<url>http://artifactory.eanadev.org/artifactory/libs-snapshot-local</url>
+		</snapshotRepository>
+	</distributionManagement>
+
+	<properties>
+		<version.java>13</version.java>
+		<version.springfox>2.9.2</version.springfox>
+		<version.metis>1.7.0</version.metis>
+		<version.lombok>1.18.12</version.lombok>
+		<version.commons.io>2.6</version.commons.io>
+		<version.maven.compiler.plugin>3.6.1</version.maven.compiler.plugin>
+		<version.coveralls>4.0.0</version.coveralls>
+		<version.surefire.plugin>2.22.1</version.surefire.plugin>
+
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	</properties>
+
+	<dependencies>
+		<!-- Spring boot -->
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-amqp</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-webflux</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
+		</dependency>
+
+		<!-- Europeana -->
+		<dependency>
+			<groupId>eu.europeana.metis</groupId>
+			<artifactId>metis-common</artifactId>
+			<version>${version.metis}</version>
+		</dependency>
+
+		<!-- Persistence -->
+		<dependency>
+			<groupId>com.h2database</groupId>
+			<artifactId>h2</artifactId>
+			<scope>runtime</scope>
+		</dependency>
```

Comment:
Oh for sure I am thinking on postgres too, this one is just for current development, since it is in memory I can go forward with development, modeling the entities that I need and then it will be as simple as switching the jpa and datasource config in the yml to a persistent database like postgres (which is the idea) but also something nice is that you can use the in memory database to use them in your testing profiles when doing integration testing